src/components/settings-row/settings-row.spec.js

```diff
@@ -77,6 +77,25 @@ describe("SettingsRow", () => {
       });
     });
 
+    describe("when headingLevel prop is provided", () => {
+      it.each([1, 2, 3, 4, 5, 6])(
+        "HTML section heading element is correct when headingLevel is %s",
+        (headingLevels) => {
+          wrapper = mount(
+            <SettingsRow
+              headingLevel={headingLevels}
+              title={title}
+              description={description}
+            >
+              Content for settings
+            </SettingsRow>
+          );
+
+          expect(wrapper.find(`h${headingLevels}`).exists()).toBeTruthy();
```

Comment:
```suggestion
          expect(wrapper.find(`h${headingLevels}`).text()).toBe(title);
```
To guarantee that the heading element is the title and not some other element, you could amend the assertion like so - this would still assert if the title is rendered in the document, but should give you reassurance that you have found the right element.